src/client/components/social-share/social-share.vue

```diff
@@ -243,49 +279,66 @@ export default {
   border-radius: 50%; 
 }
 
+/* apply shared styles to all links */
+
+#share:target .social-share__link,
 .visible {
```

Comment:
use a BEM selector for visible: `.social-share__link--visible`. Otherwise this specific styling will be applied anywhere the class `visible` is used.